src/components/PopoverMenu.tsx

```diff
@@ -193,7 +194,7 @@ function PopoverMenu({
                 {!!headerText && <Text style={[styles.createMenuHeaderText, styles.ml3]}>{headerText}</Text>}
                 {enteredSubMenuIndexes.length > 0 && renderBackButtonItem()}
                 {currentMenuItems.map((item, menuIndex) => (
-                    <MenuItem
+                    <PopoverMenuListItem
```

Comment:
the file is called `PopoverMenuItem` but the component is called `PopoverMenuListItem`.
Thats probably a mistake? I think `PopoverMenuItem` sounds good